src/host/config/IMUConfig.h

```diff
@@ -27,22 +27,37 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
+/*          horizontal top (KBox or sensor)
+              __________________
+             / o               /|
+            /                 / /
+Left Side  /     BNO055      / /   Right Side
+          /  vertical top   / /
+         /_________________/ /
+         |________________ |/
+
+       Bottom (KBox or sensor)
+*/
 
 #pragma once
 
 enum IMUMounting {
-  VerticalStbHull,
-  VerticalPortHull,
-  VerticalTopToBow,
-  //VerticalTopToStern,
-  HorizontalLeftSideToBow,
-  //HorizontalRightSideToBow
+  verticalPortHull,           // KBox mounted on port hull
+  verticalStbHull,            // KBox mounted on stb. hull
+  verticalRightSideToBow,     // KBox mounted on port hull
```

Comment:
Why have two options that are the same thing? It looks like `verticalPortHull == verticalRightSideToBow`? Or maybe I am missing something?

Reply:
Yes if I remember right, it is the same. verticalPortHull is the "deprecated" naming convention which I believe you have in your config. If you switch to the "more detailed naming convention" then we do not need it anymore.

If you like it in principle and if it is easy for you to change, please do so. 
Otherwise I think the right way is to merge your new commits, add the `[env:host36]` and delete the `verticalPortHull`, `verticalStbHull` settings in this PR, right?

Reply:
I like the new naming and I think it's good to remove the old one to keep the more explicit one. 👍